src/utils.sh

```diff
@@ -20,5 +22,25 @@ function kworkflow-help()
     "\tup,u - Wake up vm\n" \
     "\tcodestyle - Apply checkpatch on directory or file\n" \
     "\tmaintainers - Return the maintainers and the mailing list\n" \
+    "\tsearch - Search for expression on git log or directory\n" \
     "\thelp"
 }
+
+function search()
+{
+  if [[ "$#" -eq 0 ]]; then
+    complain "Expected path or 'log'"
+    exit 1
+  fi
+  if [[ "$1" == "log" ]]; then
+    git log | grep -nwI -e "$2"
+  else
```

Comment:
Maybe the "case" operator is better than the "if/else" in this case, because we want to expand this options soon.